client/src/components/App/index.js

```diff
@@ -1,12 +1,97 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from 'react-router-dom';
+import axios from 'axios';
+
 import './index.css';
+import AppBar from '../common/AppBar';
+import Limitation from '../common/limitation';
 
-function App() {
-  return (
-    <div>
-      <h1>Welcome in Applicants System</h1>
-    </div>
-  );
-}
+export default class App extends React.Component {
+  state = {
+    auth: null,
+  };
+
+  componentDidMount() {
+    axios
+      .get('/api/v1/auth')
+      .then((result) => {
+        this.setState({ ...result.data });
+      })
+      .catch(() => {
+        this.setState({ auth: false });
+      });
+  }
+
+  signupHandler = () => {
+    this.setState({
+      auth: true,
+    });
+  };
 
-export default App;
+  logoutHandler = () => {
+    axios.get('/api/v1/logout').then(() => {
+      this.setState({
+        auth: false,
+      });
+    });
+  };
+
+  render() {
+    const { auth } = this.state;
+    return (
+      <div>
+        <AppBar />
+        <Router>
+          <Switch>
+            <Route path="/404">404</Route>
+            <Route path="/500">500</Route>
```

Comment:
Add main container for all pages 
```suggestion
       <main className="container">
```